example.tex

```diff
@@ -36,8 +63,11 @@
 This text will be in the second column
 and on a second thought this is a nice looking
 layout in some cases.
-\begin{lstlisting}[language=C, showstringspaces=false]
+
+\begin{lstlisting}[language=C, showstringspaces=false, caption=C example]
+// SPDX-License-Identifier: GPL-2.0
```

Comment:
Why the license here?
If for comment highlighting, I'd suggest a comment in the function. I don't think a snippet like that is even copyrightable :-)